rustls/src/server/hs.rs

```diff
@@ -206,7 +205,7 @@ impl ExtensionProcessing {
             .is_some()
             || hello
                 .cipher_suites
-                .contains(&CipherSuite::TLS_EMPTY_RENEGOTIATION_INFO_SCSV);
+                .contains(&crate::enums::CipherSuite::TLS_EMPTY_RENEGOTIATION_INFO_SCSV);
```

Comment:
Style nit: this line is doing enough work as it is, probably better to import `CipherSuite` in some enclosing scope.